### ISSUE TITLE
this project has web site?

### ISSUE THREAD

**Issue by imkimchi** [Lymke/leagueoflegends#1]
i really want to see website of it!

**Comment by Lymke** [comment]
Yes this project has a website ! :p I'll give you the url as soon as I developp a little more features and multilingual option ;)

**Comment by Lymke** [comment]
Hi :D
http://leagueoflegends.omegatheque.com/#/champions
Soon we'll add a button "send me an email when there is promo" ;)
